lucille-core/src/main/java/com/kmwllc/lucille/stage/EmitNestedChildren.java

```diff
@@ -0,0 +1,43 @@
+package com.kmwllc.lucille.stage;
+
+
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.typesafe.config.Config;
+import java.util.Iterator;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This stage emits attached children documents, removing them from the parent document. Will skip document if no children is found.
+ *
+ * Config Parameters:
+ * - drop_parent (Boolean): if set to true, will mark parent document as dropped
+ *
+ */
+public class EmitNestedChildren extends Stage {
+
+  private final boolean dropParent;
+  private static final Logger log = LoggerFactory.getLogger(EmitNestedChildren.class);
+
+  public EmitNestedChildren(Config config) {
+    super(config, new StageSpec().withRequiredProperties("drop_parent"));
```

Comment:
we could default drop_parent to false and make it an optional property

Reply:
resolved in [a58cc54](https://github.com/kmwtechnology/lucille/pull/174/commits/a58cc54a99f2a8333a5ccd48e61575f073040bb1)